src/Parser/MistweaverMonk/Modules/Items/Velens.js

```diff
@@ -34,7 +34,25 @@ class Velens extends Module {
       return;
     }
 
-    this.healing += calculateEffectiveHealing(event, LEGENDARY_VELENS_HEALING_INCREASE);
+    // Rewriting the effectiveHealing function here, to remove the final step of removing the raw overheal from the effective healing of the trinket.
```

Comment:
Can you explain why you did this? I'd like the Velen's calculation to be consistent among all classes. This looks to be wrong as Velen's is a static 15% healing increase to the **raw** healing going out. If that increase turns out to be overhealing it is worthless, so my approach of reducing the healing gain by the overhealing really only gives you the gain in healing that did something. This ignores overhealing and acts like Velen's contributed 15% of the effective healing, which it will not have done if there was overhealing. I believe it will show too high values.